ibis/tests/expr/test_table.py

```diff
@@ -1404,3 +1405,23 @@ def test_mutate_chain():
     assert isinstance(a.op(), ops.IfNull)
     assert isinstance(b.op(), ops.TableColumn)
     assert isinstance(b.op().table.op().selections[1].op(), ops.IfNull)
+
+
+def test_multiple_dbcon():
+    """
+    Expr from multiple connections to same DB should be compatible.
+    """
+    con1 = MockBackend()
+    con2 = MockBackend()
+
+    con1.table('alltypes').union(con2.table('alltypes')).execute()
+
+
+@pytest.mark.xfail(
+    raises=RelationError, reason='Expr from different DB should fail #64'
```

Comment:
This should also resolve #64

Reply:
Cool, can you add `Closes #64` to the PR description?